ext/pgsql/tests/bug37100_9.phpt

```diff
@@ -41,9 +42,10 @@ pg_close($db);
 --CLEAN--
 <?php
 require_once('inc/config.inc');
-$db = pg_connect($conn_str);
+$table_name = 'ttable_bug37100_9';
```

Comment:
``'table_bug37100_9'``

Reply:
Fixed in https://github.com/php/php-src/pull/12608/commits/6918e6ff2af5caf8b2c01dfb2264a885399fbf34